modules/xfem/doc/content/modules/xfem/theory/theory.md

```diff
@@ -1,4 +1,127 @@
-# XFEM Theory Manual
+# XFEM Module Theory
 
-!alert construction
-The XFEM module Theory Page needs documentation.
+The MOOSE XFEM module is a library for the implementation of extended finite element (XFEM) that solve partial differential equations (PDEs) with some form of discontinuity. For an overview of the XFEM fracture capability, please see the journal article [!cite](jiang2020).
+
+## Phantom-node-based XFEM
+
+The XFEM approach as originally proposed [!cite](belytschko_elastic_1999, Dolbow99) enriches the standard continuous finite element interpolation of the solution field $\boldsymbol{u}$ as a function of spatial coordinate $\boldsymbol{x}$ and time $t$ with Heaviside and near-tip enrichment functions:
```

Comment:
In most literature, it is pointed out that the enrichment terms are added to FEM via Partition of Unity method (1996. Melenk, J. M. and Babuska, Ivo. The partition of unity finite element method basic theory and applications.) This may be a pedantic point, but I think it's worth pointing and and citing the reference.

Reply:
It should be fine if we only mention XFEM not GFEM :-)